### PR TITLE
Issues from the current milestone not matching the alias #1116

### LINKED ISSUE #1116: Issues from the current milestone not matching the alias
image

### PR DESCRIPTION
**Pull request by HansNewbie**
Fixes #1116

### CODE REVIEW

**Comment by HansNewbie** [comment]
@acruis passed tests locally.

**Comment by acruis** [comment]
Good to merge, and apologies (lol)

**Comment by HansNewbie** [comment]
@acruis no prob, my bad for not making integration test.

File: src/test/java/guitests/FilterTests.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        // test wrong alias
+        click("#dummy/dummy_col0_filterTextField");
+        selectAll();
+        type("milestone:current+s0v8f");
+        push(KeyCode.ENTER);
+        assertEquals(issuePanel.getIssueCount(), 0);
+    }
+
+
+    ////////////// NOTE: please turn off NumLock as it will interfere with this method
```
Comment (damithc) [inline src/test/java/guitests/FilterTests.java]: Write a proper javadoc for this? The `//////////////////` looks unprofessional :-)

Comment (HansNewbie) [inline src/test/java/guitests/FilterTests.java]: @damithc I will do it in a separate PR. There is also some redundancy in the method, I will abstract it out so everyone can use.

Comment (HansNewbie) [inline src/test/java/guitests/FilterTests.java]: Will be solved in #1123

**Pull request merged**